gems/actionpack/CVE-2013-4491.yml

```diff
@@ -3,17 +3,17 @@ gem: actionpack
 framework: rails
 cve: 2013-4491
 osvdb: 100528
+ghsa: 699m-mcjm-9cw8
 url: https://groups.google.com/forum/#!topic/ruby-security-ann/pLrh6DUw998
 title: Reflective XSS Vulnerability in Ruby on Rails
 date: 2013-12-03
 description: |
   There is a vulnerability in the internationalization component of Ruby on
   Rails. Under certain common configurations an attacker can provide specially
   crafted input which will execute a reflective XSS attack.
-
```

Comment:
Did `yamllint` flag this? The `|` modifier allows for multiple paragraphs along as they are indented.

Reply:
Are you commenting on the blank line that was removed?